curated_transformers/tokenization/gpt_neox_tokenizer.py

```diff
@@ -39,7 +39,11 @@ def convert_hf_tokenizer(cls: Type[Self], tokenizer: Any) -> Self:
         deserialized = json.loads(serialized)
         vocab_merges = deserialized["model"]
         merges = [tuple(merge.split(" ")) for merge in vocab_merges["merges"]]
-        processor = ByteBPEProcessor(vocab_merges["vocab"], merges)
+        vocab = vocab_merges["vocab"]
```

Comment:
Don't we need to add this logic to other `ByteBPETokenizer` subclasses as well?

Reply:
Definitely. I want to do this in a follow up PR, where we also switch to `tokenizer.json` for the non-sentencepiece tokenizers.